CITATION.cff

```diff
@@ -0,0 +1,110 @@
+  type: article
+  year: 2021
+  journal: Global Change Biology
+  URL: https://onlinelibrary.wiley.com/doi/10.1111/gcb.15943
+	publisher: Wiley Online Library
+  doi: 10.1111/gcb.15943
+  authors:
+  - family-names: "Badgley"
+    given-names: "Grayson"
+    orcid: https://orcid.org/0000-0003-1011-4573
+  - family-names: "Freeman"
+    given-names: "Jeremy"
+    orcid: https://orcid.org/0000-0001-7077-7972
+  - family-names: "Hamman"
+    given-names: "Joseph J."
+    orcid: https://orcid.org/0000-0001-7479-8439
+  - family-names: "Haya"
+    given-names: "Barbara"
+    orcid: https://orcid.org/0000-0002-7010-3908
+  - family-names: "Trugman"
+    given-names: "Anna T."
+    orcid: https://orcid.org/0000-0002-7903-9711
+  - family-names: "Anderegg"
+    given-names: "William R.L."
+    orcid: https://orcid.org/0000-0001-6551-3331
+  - family-names: "Cullenward"
+    given-names: "Danny"
+    orcid: https://orcid.org/0000-0002-6803-9572
+
+references:
+  - description: "A repository containing the input, intermediate, and final output data files underlying the analysis and figures in Badgley et al., “Systematic over-crediting in California’s forest carbon offsets program”."
+    type: doi
+    value: https://zenodo.org/record/5565457#.YZL1Z1NlBpR
+  authors:
+  - family-names: "Badgley"
+    given-names: "Grayson"
+    orcid: https://orcid.org/0000-0003-1011-4573
+  - family-names: "Freeman"
+    given-names: "Jeremy"
+    orcid: https://orcid.org/0000-0001-7077-7972
+  - family-names: "Hamman"
+    given-names: "Joseph J."
+    orcid: https://orcid.org/0000-0001-7479-8439
+  - family-names: "Haya"
+    given-names: "Barbara"
+    orcid: https://orcid.org/0000-0002-7010-3908
+  - family-names: "Trugman"
+    given-names: "Anna T."
+    orcid: https://orcid.org/0000-0002-7903-9711
+  - family-names: "Anderegg"
+    given-names: "William R.L."
+    orcid: https://orcid.org/0000-0001-6551-3331
+  - family-names: "Cullenward"
+    given-names: "Danny"
+    orcid: https://orcid.org/0000-0002-6803-9572
+
+references:
```

Comment:
slightly concerned about the duplicated `references` key here. 

Can you try running this validator on the file and reporting back what it says:

```
pip install cffconvert

# Validate your CFF file
cffconvert --validate
```